### PR TITLE
Phobia trigger words are highlighted in pulsing red/black

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
tweak: Phobia trigger words are now more noticeable.
/:cl:

Spooky

### CODE REVIEW

**Comment by Arianya** [comment]
How does it pulse? Screenshot/gif needeeeed

**Comment by XDTM** [comment]
It's text that shifts from black to red and back in 0.75 seconds

Can't compile code to get gifs at the moment

**Comment by AnturK** [comment]
It's just css, https://jsfiddle.net/kveyfurh/

File: code/modules/goonchat/browserassets/css/browserOutput.css [hunk 1]
```diff
@@ -398,7 +398,15 @@ h1.alert, h2.alert		{color: #000000;}
...
-		75% { color: #410194; }
+	75% { color: #410194; }
+	100% { color: #3bb5d3; }
+}
+
+.phobia			{color: #dd0000;	font-weight: bold; animation: phobia 750ms infinite;}
+	@keyframes phobia {
+		0% { color: #0d0d0d; }
+		50% { color: #dd0000; }
+		100% { color: #0d0d0d; }
```
Comment (SpaceManiac) [inline code/modules/goonchat/browserassets/css/browserOutput.css]: The indentation here is wacky and there's some spurious line changes. Ditto below.

File: code/datums/brain_damage/phobia.dm [hunk 2]
```diff
@@ -71,19 +71,20 @@
 	if(!owner.can_hear() || world.time < next_scare) //words can't trigger you if you can't hear them *taps head*
 		return message
 	for(var/word in trigger_words)
-		var/reg = regex("(\\b|\\A)[REGEX_QUOTE(word)]'?s*(\\b|\\Z)", "i")
-
+		var/regex/reg = regex("(\\b|\\A)[REGEX_QUOTE(word)]'?s*(\\b|\\Z)", "i")
+		
 		if(findtext(raw_message, reg))
 			addtimer(CALLBACK(src, .proc/freak_out, null, word), 10) //to react AFTER the chat message
 			break
```
Comment (SpaceManiac) [inline code/datums/brain_damage/phobia.dm]: Doesn't this `break` mean the replacement below never runs?

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Comment by Arianya** [comment]
Looks pretty fancy.

**Comment by Militaires** [comment]
make it the orange TRIGGERED meme

**Comment by SpaceManiac** [comment]
Awaiting author reply for 7+ days, closed as stale, contact a maintainer for reopen

**Comment by XDTM** [comment]
@SpaceManiac reviews addressed in new commits

**Comment by XDTM** [comment]
Reviews addressed in new commits

**Pull request merged**